compiler/z/codegen/OMRMachine.cpp

```diff
@@ -1108,7 +1108,7 @@ OMR::Z::Machine::findBestLegalEvenRegister(uint64_t availRegMask)
          }
 
       lastEvenReg = _registerFile[i];
-      //_cg->traceRegWeight(lastEvenReg, lastEvenReg->getWeight());
+      //self()->cg()->traceRegWeight(lastEvenReg, lastEvenReg->getWeight());
```

Comment:
Likely unrelated to this change, but is there a particular reason this line is commented out? There are more instances of this bellow.

Reply:
I don't know the exact reason, but I imagine these statements were introduced to help diagnose a problem and were never removed.  Presumably, the developer also thought the compile-time overhead of leaving them in the code was too high.  Addressing this is beyond the scope of this PR but likely something to look into in the future.